lecture-notes/4-cor.md

```diff
@@ -692,3 +735,48 @@ their releases against their customers' tests.
 
 What do you do if you find breaking behavior surfaced by a large integration
 test written by your client?
+
+## Lecture 4.5
+
+### Strategies for isolating units
```

Comment:
If we talk about mocking here, let's speak critically about it. It has its uses, but it's not a magic bullet---and in Tom's opinion, most real-world implementations are quite lacking and often require the programmer to make non-test code harder to read.